examples/README.md

```diff
@@ -0,0 +1,54 @@
+# examples
+
+This directory contains example code for both installing Contour Envoy.
+ also for setting up various extensions and other pieces of software that may help with running Contour.
+
+Most of the directories contain a complete set of Kubernetes YAML that can be applied to a cluster. This section describes the purpose of each directory, and inside each directory is a README.md that describes the details.
```

Comment:
s/Most of the directories/Most subdirectories/

Reply:
newline after period

s/each directory/each subdirectory (the opening sentence described this as the directory, so the children must be subdirectories)

Reply:
Drop , and inside each directory is a README.md that describes the details.